components/ILIAS/Administration/classes/class.ilAdministrationSettingsFormHandler.php

```diff
@@ -179,13 +179,10 @@ public static function addFieldsToForm(
 
         // cron jobs - special handling
 
-        $parent_gui = new ilObjSystemFolderGUI(null, SYSTEM_FOLDER_ID, true);
-        $parent_gui->setCreationMode(true);
-
-        $gui = new ilCronManagerGUI();
+        $gui = ilObjCronGUI::create();
```

Comment:
Can we move this inside the `if` block?

Reply:
No, the `if` tests for `$data` which is returned from the `$gui` or do you mean another `if`? (But I can change `is_array($data) && count($data)` to `$data !== []` in any case).

Reply:
Sorry, I did not see the usage :facepalm: